dnf.spec

```diff
@@ -198,56 +221,64 @@ Requires:       rpm-plugin-systemd-inhibit
 %else
 Recommends:     rpm-plugin-systemd-inhibit
 %endif
-# dnf-langpacks package is retired in F25
-# to have clean upgrade path for dnf-langpacks
-Obsoletes:      python3-dnf-langpacks < %{dnf_langpacks_ver}
 
 %description -n python3-%{name}
 Python 3 interface to DNF.
 %endif
 
 %package automatic
-Summary:        Alternative CLI to "dnf upgrade" suitable for automatic, regular execution.
+Summary:        %{pkg_summary} - automated upgrades
```

Comment:
it doesn't handle only upgrades

Reply:
Additionally it prepares packages for upgrade, or reports about available upgrades, but this is a detail.